eventmesh-connectors/eventmesh-connector-rocketmq/src/main/resources/source-config.yml

```diff
@@ -26,7 +26,7 @@ pubSubConfig:
     passWord: rocketmqPassWord
 connectorConfig:
     connectorName: rocketmqSource
-    nameserver: 127.0.0.1:9877
+    nameserver: 192.168.123.50:9876
```

Comment:
Using 127.0.0.1:9876 might be a better choice

Reply:
Yes, the original port was mistaking.